DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces_overview/mm-host/src/vector_add.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+
+// oneAPI headers
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+
+// Buffer locations for MM Host interfaces
+static constexpr int BL0 = 0;
+static constexpr int BL1 = 1;
+static constexpr int BL2 = 2;
+
+// Forward declare the kernel name in the global scope. This is an FPGA best
+// practice that reduces name mangling in the optimization reports.
+class SimpleVAdd;
+
+struct SimpleVAddKernel {
+  sycl::ext::oneapi::experimental::annotated_arg<
+      int *, decltype(sycl::ext::oneapi::experimental::properties{
+                 sycl::ext::intel::experimental::buffer_location<BL0>,
+                 sycl::ext::intel::experimental::dwidth<32>,
+                 sycl::ext::intel::experimental::latency<0>,
+                 sycl::ext::intel::experimental::read_write_mode_read,
+                 sycl::ext::intel::experimental::wait_request_requested})>
```

Comment:
```suggestion
                 sycl::ext::intel::oneapi::buffer_location<BL0>,
                 sycl::ext::intel::oneapi::dwidth<32>,
                 sycl::ext::intel::oneapi::latency<0>,
                 sycl::ext::intel::oneapi::read_write_mode_read,
                 sycl::ext::intel::oneapi::wait_request_requested})>
```

Reply:
with trunk/20230816, `sycl::ext::intel::experimental::` seems to be the only namespace that compiles

Reply:
then check with engineering on what it is supposed to be; yohann and i tried compiling this with 2024.0 and it only worked with `oneapi` today. we were using sycl/trunk/202306XX (the qualified SYCL)

Reply:
Brox confirmed the previous namespace `sycl::ext::oneapi::experimental` for the different properties was an error. Should be `ext::intel::experimental` going forward